lsp-ui.el

```diff
@@ -1,12 +1,12 @@
-;;; lsp-ui.el --- UI modules for lsp-mode            -*- lexical-binding: t; -*-
+;;; lsp-ui.el --- UI modules for lsp-mode -*- lexical-binding: t; -*-
 
 ;; Copyright (C) 2017 Tobias Pisani
 ;; Copyright (C) 2018 Sebastien Chapuis, Fangrui Song
 
 ;; Author: Sebastien Chapuis <sebastien@chapu.is>, Fangrui Song <i@maskray.me>
-;; Keywords: lsp
+;; Keywords: languages, tools
 ;; URL: https://github.com/emacs-lsp/lsp-ui
-;; Package-Requires: ((emacs "25.1") (dash "2.14") (dash-functional "1.2.0") (lsp-mode "6.0") (markdown-mode "2.3"))
+;; Package-Requires: ((emacs "26.1") (dash "2.14") (dash-functional "1.2.0") (lsp-mode "6.0") (markdown-mode "2.3"))
```

Comment:
Is it necessary to drop emacs25 support ?

Reply:
when using `package-lint` it warns me that we use some function that I don't record now that is not supported on emacs 25, I'll find out what function was

Reply:
when I run `package-lint-current-buffer` on `lsp-ui-sideline.el`
![image](https://user-images.githubusercontent.com/7820865/80887772-94570200-8c94-11ea-9f7b-bc90a3314fc3.png)

Reply:
`lsp-ui.el`
```
430:33: error: You should depend on (emacs "26.1") if you need `frame-outer-width'.
432:37: error: You should depend on (emacs "26.1") if you need `frame-outer-width'.
535:15: error: You should depend on (emacs "26.1") if you need `line-number-display-width'.
657:33: error: You should depend on (emacs "26.1") if you need `frame-parent'.
840:21: error: You should depend on (emacs "26.1") if you need `frame-parent'.
```

Reply:
What do you think @brotzeit ?

Reply:
So actually lsp-ui already depends on emacs26 ? Hmmm, IMO we should stick with the dependency of lsp-mode. But it seems too late already =)

I guess most lsp-ui users want fancy stuff and use new emacs versions anyway...
@yyoncho  What do you think ?

Reply:
AFAIK all functionality has emacs25 fallback(or at least used to have). But IMO since most of it relies on emasc 26, either way, it makes sense to introduce emacs 26+ dependency.